ipatests/test_integration/test_installation.py

```diff
@@ -196,3 +197,36 @@ def test_install_master(self):
 
     def test_install_kra(self):
         tasks.install_kra(self.master, first_instance=True)
+
+
+@pytest.mark.ds_acceptance
+class TestDSAcceptance(IntegrationTest):
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True)
+
+    def test_install_replica0(self):
+        tasks.install_replica(self.master, self.replicas[0], setup_ca=False,
+                              setup_kra=False, setup_dns=False)
+
+
+@pytest.mark.cs_acceptance
+class TestCSAcceptance(IntegrationTest):
```

Comment:
test steps:

- master with ca and kra, no dns
- replica0 caless, no dns
- replica0 install ca
- replica0 install kra
- replica0 install dns
- replica1 with CA, no dns
- replica1 install kra
- replica1 install dns
- replica2 install CA + KRA  # this is more or less the same as replica 1 because now kra is supported only as two step installation (maybe we can remove this test)
- replica2 install dns